app/frontend/src/features/auth/useAuthStyles.ts

```diff
@@ -1,11 +1,12 @@
 import { makeStyles } from "@material-ui/core";
 
-import AuthBg from "./resources/auth-bg.png";
+import DesktopAuthBg from "./resources/desktop-auth-bg.png";
+import MobileAuthBg from "./resources/mobile-auth-bg.png";
 
 const useAuthStyles = makeStyles((theme) => ({
   backgroundBlurImage: {
     backgroundColor: "#f3f3f3",
```

Comment:
`theme.palette.gray[x]`

Reply:
How does it work? Should I add new grey color there? What number to use?

Reply:
Whichever fits the closest